src/scripts/core/cartesian.js

```diff
@@ -147,8 +147,8 @@
                         var em = regularXBounds.height;
                         var ang = xOptions.labels && xOptions.labels.rotation ? xOptions.labels.rotation % 360 : 0;
                         var xLabelHeightUsed = ang === 0 ? regularXBounds.height : Math.ceil(Math.abs(xLabelBounds.width * Math.sin(_.nw.degToRad(ang))) + em / 5) ;
-                        this.options.chart.internalPadding.bottom = this.options.chart.padding.bottom ||
-                            maxTickSize(this.options.xAxis) + (this.options.xAxis.tickPadding || 0) +
+                        this.options.chart.internalPadding.bottom = maxTickSize(this.options.xAxis) + 
```

Comment:
This needs to include `this.chart.padding.bottom ||`, this is so the padding can be overridden by setting it manually if not there's not way to force it to be something.